steam/ext/dota2/protobufs/common.py

```diff
@@ -0,0 +1,100 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
+# sources: dota_gcmessages_common.proto
+# plugin: python-betterproto
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+import betterproto
+
+from ....protobufs.msg import GCProtobufMessage
+from ..enums import EMsg
+
+
+# fmt: off
+class CMsgDOTAProfileCardEStatID(betterproto.Enum):
+    Wins               = 3
+    Commends           = 4
+    GamesPlayed        = 5
+    FirstMatchDate     = 6
+    PreviousSeasonRank = 7
+    GamesMVP           = 8
+# fmt: on
```

Comment:
betterproto enums don't need formatting like the regular ones because they shouldn't be exposed
```suggestion
class CMsgDOTAProfileCardEStatID(betterproto.Enum):
    Wins               = 3
    Commends           = 4
    GamesPlayed        = 5
    FirstMatchDate     = 6
    PreviousSeasonRank = 7
    GamesMVP           = 8
```